File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
    let temp = []

    for (i = 0; i < nums.length; i++) {
        if (!temp.includes(nums[i])) {
            temp.push(nums[i])
        }
    }
    console.log(temp)
    for (j = 0; j < temp.length; j++) {
        nums[j] = temp[j];
    }
    return temp.length
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int toPlace = 0;
        int i = 1;

        for (i; i < nums.size(); i++) {
            if (nums[i] != nums[i - 1]) {
                nums[toPlace++] = nums[i - 1];
            }
        }
        nums[toPlace++] = nums.back();

        return toPlace;
    }
};